packages/insured-bridge-relayer/src/RelayerHelpers.ts

```diff
@@ -0,0 +1,47 @@
+import Web3 from "web3";
```

Comment:
I added this set of helpers to do some common things the relayer will need to do. in particular, see the `approveL1Tokens` method